web/README.md

```diff
@@ -108,7 +113,52 @@ python manage.py test
 https://www.onlinetutorialspoint.com/django/django-model-view-template-mvt-overview.html
 
 ## Folder and file structure
-Where is what, and what does it do.
+
+As of now, the project consists of 6 folders, with 4 of them being django apps
+(`adminpage`, `authenticate`, `happenings` and `swiping`),
+one being a folder for util files (`util`) and the last one being the main folder (`web`).
+
+####Adminpage
```

Comment:
Need a whitespace between `#` and characters to render the headers properly. Also I think we should use `###` instead of `####` following this [rule](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md001---heading-levels-should-only-increment-by-one-level-at-a-time)